pkg/asset/machines/master.go

```diff
@@ -52,20 +53,33 @@ import (
 type Master struct {
 	UserDataFile       *asset.File
 	MachineConfigFiles []*asset.File
+	SecretFiles        []*asset.File
```

Comment:
I think we should separate this out to a different asset... 

Or alteast make it clear that this is only for bare-metal...

Reply:
I did think about that, but the host assets require information available in the computed Machine assets in order to link the two together and avoid reprovisioning the master hosts.

I couldn't figure out if there is some way to say that when the platform is baremetal we need to add these extra things and make the machines available to the function that computes them. I could say that the Baremetal asset depends on the Machine assets, but it wasn't clear that that wouldn't just end up computing the machines twice. Is that what dependencies.Get() does for us?

What is a good way to make it more clear these are only used for baremetal? Renaming the fields? Adding them to a sub-struct? Comments?

Reply:
can we start by separating them by new line in the struct and commenting...

Reply:
Sure, a comment is easy.